htdocs/class/xoopsform/formlabel.php

```diff
@@ -64,6 +64,10 @@ public function getValue($encode = false)
      */
     public function render()
     {
-        return $this->getValue();
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            return "<div class='form-group form-inline'>" . $this->getValue() . "</div>";
+        } else {
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->